app/talk/comment-report-form.cjsx

```diff
@@ -1,26 +1,44 @@
-React = require 'react'
+React = {findDOMNode} = require 'react'
 Feedback = require './mixins/feedback'
+talkClient = require '../api/talk'
 
 module?.exports = React.createClass
   displayName: 'TalkCommentReportForm'
 
   mixins: [Feedback]
 
+  propTypes:
+    comment: React.PropTypes.object
+    error: ''
```

Comment:
Oops that was my bad, I meant for that to be in `getInitialState`. Will push an update in a sec. 

I like propTypes quite a bit for anything intended to be used in multiple places, mainly to make it easy for people to know that components have everything they need, and for a sort of documentation